os/linux/gbmbufferhandler.cpp

```diff
@@ -239,9 +239,9 @@ void *GbmBufferHandler::Map(HWCNativeHandle handle, uint32_t x, uint32_t y,
 #endif
 }
 
-void GbmBufferHandler::UnMap(HWCNativeHandle handle, void *map_data) {
+int32_t GbmBufferHandler::UnMap(HWCNativeHandle handle, void *map_data) {
   if (!handle->bo)
-    return;
+    return -1;
 
   return gbm_bo_unmap(handle->bo, map_data);
```

Comment:
gbm_bo_unmap returns void. [Check here](https://github.com/intel/minigbm/blob/master/gbm.h#L343)

Returning void in a function returning int32_t is not allowed. We just return the success code after calling gbm_bo_unmap.